src/test/java/teammates/test/cases/action/InstructorFeedbackQuestionAddActionTest.java

```diff
@@ -674,7 +669,21 @@ public void testExecuteAndPostProcessRubricQuestion() {
                 Const.ParamsNames.FEEDBACK_QUESTION_EDITTYPE, "edit",
         };
 
-        InstructorFeedbackQuestionAddAction action = getAction(params);
+        ______TS("Typical case with attached weights");
+
+        // tests when descriptions are missing as well.
+
+        String[] params = new String[] {
+                Const.ParamsNames.FEEDBACK_QUESTION_NUMBER, "1",
+                Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_WEIGHTS_ASSIGNED, "on",
+                Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_WEIGHT + "-0", "-1",
+                Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_WEIGHT + "-1", "2",
+        };
+
+        List<String> requestedParams = new ArrayList<>(Arrays.asList(requiredParams));
+        Collections.addAll(requestedParams, params);
+
+        InstructorFeedbackQuestionAddAction action = getAction(requestedParams.toArray(new String[0]));
```

Comment:
Why not just `requestedParams.toArray()`? There are multiple places of this problem in your PR.

> Note that toArray(new Object[0]) is identical in function to toArray(). -- from java8 doc

Reply:
`.toArray()` returns an `object[]`, and `getAction` method takes `String...` as arguments. It throws an error if I just use `requestedParams.toArray()`

![screenshot 497](https://user-images.githubusercontent.com/10681755/40887886-eb34af22-676c-11e8-9c91-2a6616d268c8.png)

Reply:
I see